templates/uVision.adapter.json

```diff
@@ -0,0 +1,15 @@
+{
+    "tasks": [{
+        "label": "CMSIS Load+Run",
+        "type": "shell",
+        "command": "<%= uv4 %>",
+        "options": { "cwd": "${workspaceFolder}" },
```

Comment:
Should we add the `/<%= solution_folder %>` folder here, like we do everywhere else? 
uVision does not have the notion of a separate workspace - the directory of the csolution.yml is the workspace folder (I think).